security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -329,11 +335,10 @@ func (sc *SecretController) upsertSecret(saName, saNamespace string) {
 			saNamespace, GetSecretName(saName), err)
 		return
 	}
-	rootCert := sc.ca.GetCAKeyCertBundle().GetRootCertPem()
```

Comment:
Can we use the root cert in CAKCB as the source of truth for multiple trust anchors? It should be kept in-sync with the K8s ConfigMap by the configmap.controller.